docs/guides/connecting_to_your_data/cloud/s3/spark.md

```diff
@@ -11,13 +11,13 @@ import SparkAdditionalNotes from '../../components/spark_additional_notes.md'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This guide will help you connect to your data stored on AWS S3 using spark.
+This guide will help you connect to your data stored on AWS S3 using Spark.
 This will allow you to validate and explore your data.
 
 <Prerequisites>
 
 - Have access to data on an AWS S3 bucket
-- Have a access to a working spark installation
+- Have a access to a working Spark installation
```

Comment:
while we're in here, should be "have access"

Reply:
Oops :) Sorry about this

Reply:
```suggestion
- Have access to a working Spark installation
```